spec/routing/comments_routing_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "rails_helper"
+
+RSpec.describe CommentsController, type: :routing do
+  describe "routing" do
+
+    it "routes to #index" do
+      expect(:get => "/comments").to route_to("comments#index")
+    end
+
+    it "routes to #new" do
+      expect(:get => "/comments/new").to route_to("comments#new")
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)